azure-pipelines.yml

```diff
@@ -11,8 +11,6 @@ variables:
   _SignType: real
   _TeamName: DotNetSpark
   MSBUILDSINGLELOADCONTEXT: 1
-  ArtifactPath: '$(Build.ArtifactStagingDirectory)\Microsoft.Spark.Binaries'
```

Comment:
I think this is used for internal build path?

Reply:
good catch.  readded